Firmware/Marlin_main.cpp

```diff
@@ -3469,8 +3476,21 @@ static void gcode_M600(const bool automatic, const float x_position, const float
     
         // Feed a little of filament to stabilize pressure
         if (!automatic) {
-            current_position[E_AXIS] += FILAMENTCHANGE_RECFEED;
-            plan_buffer_line_curposXYZE(FILAMENTCHANGE_EXFEED);
+            if (isPrintPaused)
+            {
+                // Return to retracted state during a pause
+                current_position[E_AXIS] -= default_retraction;
+                plan_buffer_line_curposXYZE(FILAMENTCHANGE_RFEED);
```

Comment:
Not sure if this is actually required.

I suspect the E-move later will overwrite this:

```
       // Set E position to original
        plan_set_e_position(lastpos[E_AXIS]);
    
        memcpy(current_position, lastpos, sizeof(lastpos));
        set_destination_to_current();
```